CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerFeedbackTests.m

```diff
@@ -388,6 +422,89 @@ - (void)testReadyToSendOnBidStateOnEmptyBid {
   XCTAssertEqual([self.lastSentMessages count], 1);
 }
 
+#pragma mark - Ready to send management (Live Bidding)
+
+- (void)testReadyToSendOnLiveBidValidBidReceived {
+  self.cdbResponse.cdbBids = @[ self.validBid ];
+  [self configureApiHandlerMockWithCdbRequest:self.cdbRequest
+                                  cdbResponse:self.cdbResponse
+                                        error:nil];
+
+  [self fetchLiveBidForAdUnit:self.adUnit];
+
+  XCTAssertEqual(self.feedbackFileManagingMock.readWriteDictionary.count, 0);
+  XCTAssertEqual(self.lastSentMessages.count, 1);
+}
+
+- (void)testReadyToSendOnLiveBidNoBidReceived {
```

Comment:
Isn't this a dup of `testReadyToSendOnLiveBidStateOnEmptyBid` ?
I think that this sentence make more sense: no bid over empty bid

Reply:
Done